packages/itwin/property-grid/src/test/components/PropertyGridContent.test.tsx

```diff
@@ -108,6 +108,25 @@ describe("<PropertyGridContent />", () => {
     expect(spy).to.be.calledOnce;
   });
 
+  it("renders with extended props action buttons", async () => {
+    const imodel = {} as IModelConnection;
+    const fake = sinon.fake();
+
+    const renderTestActionButton = (dataProvider: IPresentationPropertyDataProvider) => {
+      fake(dataProvider);
+      return <div>Test action button</div>;
+    };
+
+    const { getAllByText } = renderWithContext(
+      <PropertyGridContent dataProvider={provider} imodel={imodel} actionButtonRenderers={[({ dataProvider }) => renderTestActionButton(dataProvider)]} />,
```

Comment:
Why not `actionButtonRenderers={[fake]}`?

Reply:
Could do that, though we have no test that check if actionButtons adds element to the DOM. Thought it might be useful to have one.

Reply:
In that case you could do:

```tsx
const fake = sinon.fake().returns(<div>Test action button</div>);
// or if fake does not support returns
const stub = sinon.stub().returns(<div>Test action button</div>);
```

Reply:
yeah, I think it's either 
```ts
const fake = sinon.fake(() => {
  return <div>Test action button</div>;
});
```

or
```ts
const stub = sinon.stub().returns(<div>Test action button</div>);
```

I also think this stub/fake should be passed straight as an action button renderer, rather than called from one, e.g. `actionButtonRenderers={[fake]}`. Then you can check for correct call with `expect(fake).to.be.calledWith(sinon.match((arg) => arg.dataProvider === provider))`

Reply:
adjusted.